src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java

```diff
@@ -134,28 +134,28 @@ public void init(FMLInitializationEvent event) {
 
 		// Counter
 		if (Loader.isModLoaded("gregtech")) {
-			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockCounter), "SSS", "CBC", "SSS", 'B', Blocks.chest, 'S', "plateStone", 'C',"plateWood"));
+			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockCounter), "SSS", "CBC", "SSS", 'B', Blocks.chest, 'S', "plateStone", 'C', "plateWood"));
 		} else {
 			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockCounter), "SSS", "CBC", "CCC", 'B', Blocks.chest, 'S', Blocks.stone, 'C', Blocks.hardened_clay));
 		}
 
 		// Counter Corner
 		if (Loader.isModLoaded("gregtech")) {
-			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockCounterCorner), "SSS", "BCC", "SSS", 'B', Blocks.chest, 'S',"plateStone", 'C', "plateWood"));
+			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockCounterCorner), "SSS", "SIS", "SS ", 'I', "screwIron", 'S', "plateStone"));
```

Comment:
Electro Bots Suggestion was different. Why not using this recipe ? I am fine to drop the chest

Reply:
So that it looks more like the block. The upper one doesn't have wood showing, and the lower one has an L-shape of wood for the back of the counter.